cowprotocol/accounting/slippage/slippage_prices_4064601.sql

```diff
@@ -0,0 +1,131 @@
+-- This query computes prices for use in slippage on CoW Protocol
+--
+-- It gives hourly prices for all tokens transferred to or from the protocol.
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
+-- - hour: hour for which a price is valid
+-- - token_address: address of token with a price. contract address for erc20 tokens,
+--   0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee for native token
+-- - decimals: decimals of the token, can be null
+-- - price: USD price of one unit (i.e. pow(10, decimals) atoms) of a token
+-- - price_atom: USD price of one atom (i.e. 1. / pow(10, decimals) units) of a token
+
+with token_times as (
+    select
+        token_address,
+        date_trunc('hour', block_time) as hour --noqa: RF04
+    from "query_4021257(blockchain='{{blockchain}}',start_time='{{start_time}}',end_time='{{end_time}}')"
+    where token_address != 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+    group by 1, 2
+),
+
+precise_prices as (
```

Comment:
maybe let's add some comments to describe what each of the following tables computes